ludwig/schema/model_types/utils.py

```diff
@@ -421,9 +427,35 @@ def _set_mixtral_target_modules(config: "ModelConfig") -> None:
     if config.adapter.type != "lora" or config.adapter.target_modules:
         return
 
-    logger.info("Setting adapter target modules to ['q_proj', 'v_proj'] for Mixtral 7x8 base model with LoRA adapter.")
+    target_modules = ["q_proj", "v_proj"]
+
+    logger.info(f"Setting adapter target modules to {target_modules} for Mixtral 7x8 base model with LoRA adapter.")
+    config.adapter.target_modules = target_modules
+
+
+def _set_phi2_target_modules(config: "ModelConfig") -> None:
+    """If the base model is Phi-2, LoRA is enabled and the target modules are not set, set the target modules to
+    maximize performance."""
+    if config.base_model not in {
+        "susnato/phi-1_dev",
+        "susnato/phi-1_5_dev",
+        "susnato/phi-2",
+        "microsoft/phi-1",
+        "microsoft/phi-1_5",
+        "microsoft/phi-2",
```

Comment:
I feel like I've seen this list in other places.

Is it worth it to make this in to a shared map and do `.keys()`?

Reply:
Yeah I can do that